homeworks/taron_hakobyan/Project1.java

```diff
@@ -0,0 +1,30 @@
+
+import java.util.Scanner;
+public class Project1{
```

Comment:
lets name classes more specifically and also add javadocs to describe the problem you're trying to solve